plugins/jabra-file/fu-jabra-file-device.c

```diff
@@ -56,20 +53,28 @@ fu_jabra_file_device_probe(FuDevice *device, GError **error)
 				    "update interface not found");
 		return FALSE;
 	}
-	for (guint8 i = 0; i < ifaces->len; i++) {
+	for (guint i = 0; i < ifaces->len; i++) {
+		g_autoptr(GPtrArray) endpoints = NULL;
+		FuUsbEndpoint *ep1 = NULL;
+		FuUsbEndpoint *ep2 = NULL;
 		iface = g_ptr_array_index(ifaces, i);
 		if (fu_usb_interface_get_class(iface) == FU_USB_CLASS_HID) {
 			endpoints = fu_usb_interface_get_endpoints(iface);
-			if (NULL == endpoints || endpoints->len == 0)
+			if (NULL == endpoints || endpoints->len == 0 || endpoints->len < 2)
```

Comment:
Coverity isn't going to like the duplicate logic -- remove `endpoints->len == 0`